### DISCUSSION TITLE
Easy full body tracking experiment

### DISCUSSION THREAD

**Discussion by davidak** [matrix-org/thirdroom#343] [Ideas]
You can have a 3D VRM Avatar and full body tracking on https://3d.kalidoface.com/ in the browser. It's works quite well.

It is open source: https://github.com/yeemachine/kalidokit

You can probably quite easily copy and paste the VRM avatar and full body tracking code into thirdroom and see how well it works.

**Comment by lanmower** [comment]
I tried kalidokit in playcanvas with a readyplayer.me mesh have the project there somewhere, had problems with inverted arm rotation.

**Reply by lanmower** [reply]
I also managed to improve its framerate drastically by updating the mediapipe it usese
